flow/e2e/clickhouse/peer_flow_ch_test.go

```diff
@@ -474,9 +475,7 @@ func (s ClickHouseSuite) Test_Weird_Table_And_Column() {
 	`, srcFullName))
 	require.NoError(s.t, err)
 
-	_, err = s.Conn().Exec(context.Background(), fmt.Sprintf(`
-	INSERT INTO %s (key) VALUES ('init');
-	`, srcFullName))
+	_, err = s.Conn().Exec(context.Background(), fmt.Sprintf("INSERT INTO %s (key) VALUES ('init')", srcFullName))
```

Comment:
can you name the table `c0mPlex_table-nAm3`, with one column called `c0mplex_column-nAm3` please?

Reply:
No, hyphens in column names don't work. Will try hyphen in table name once tests are passing

Reply:
tested table name, it also has avro choking